File: Build_IT_DataAccess/DeadLoads/Models/Material.cs
using Build_IT_Data.Units.Enums;

namespace Build_IT_DataAccess.DeadLoads.Models
{
    public class Material
    {
        #region Properties
        
        public long Id { get; set; }
        public string Name { get; set; }
        public double MinimumDensity { get; set; }
        public double MaximumDensity { get; set; }
        public LoadUnit Unit { get; set; }
        public string DocumentName { get; set; }
        public string Comments { get; set; }
        public Subcategory Subcategory { get; set; }
        public long SubcategoryId { get; set; }

        #endregion // Properties
    }
}

File: Build_IT_Web_old/ClientApp/app/modules/loads/models/dead-loads/interfaces/material.ts
import { LoadUnit } from "../enums/loadUnit";
import { Addition } from "./addition";

export interface Material {
    name: string;
    maximumDensity: number;
    minimumDensity: number;
    unit: LoadUnit;
    documentName: string;
    comments: string;

    additions: Addition[];
}